src/sdk/python/rtdip_sdk/pipelines/_pipeline_utils/weather.py

```diff
@@ -0,0 +1,60 @@
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pyspark.sql.types import StructType, StructField, DoubleType, StringType, IntegerType
+
+WEATHER_FORECAST_SCHEMA = StructType(
```

Comment:
Can we keep these names cased. Such as Class, ExpireTimeGMT, FcstValid to be consistent across the models?